### ISSUE TITLE
Easy Mode Gyms

### ISSUE THREAD

**Issue by Archenemis** [ObliqueNET/Stratos#258]
A way to help get new players and more casual players to get into battling and trying the gyms out.  Probably already been suggested before.. but I have a bit of a twist on it to help keep from splitting players or having everyone just run all the Easy Mode stuff.

Essentially introduce an NPC Gym Leader to each of the Gym Locations that players can challenge. Have the pokemon become level locked to the levels required and perhaps even restrict it so that some pokes can't be used (Like Legends and Ultra Beasts and pokemon that can't naturally be the level that they are for the level limit of the gym.) The Gym leader will use a fairly tough team, but one that ultimately shouldn't be "Too difficult" to beat. If possible, give them "Better AI" Than your normal NPC's.. or at least some way to guess what the opponent will likely be doing next.

Essentially by doing this, hoping to get players to build up their teams and encourage them to try the Gyms out.

The twist on this.. to keep players from just going down the line and spamming all the gym leaders or to just beat only the NPC gym leaders and be done with it. First, Losing puts a cooldown on them from challenging any of the NPC Gym leaders (And maybe the actual Gym Leaders) for.. say.. 1-6 hours. This makes it so that people can't just keep spamming them and hogging up the NPC's, and promotes them to actually think and plan out their teams a little. 

The other thing I'd suggest is that Players will essentially hit a wall on the Easy Mode Gyms. Making it so that they could.. say beat the Electric and (Maybe) Poison Gyms at first without restriction. But in order to challenge the ground Gym, You have to go back and face the Electric Gym on Normal Mode to face the next Easy Mode one.

But.. like I said.. This "Should" help encourage those who are iffy on the gyms or shy about them to give it a shot.

**Comment by Archenemis** [comment]
Oh.. Side note.. Rewards wise.. I'd say don't reward them with a Badge until the Normal Mode is beaten.. the reward for beating Easy Mode should be rather light compared.

**Comment by ThaGreenDuck** [comment]
Ohhh boiii! You have been busy with all of your idea's!

I don't think there has to come a easy mode in gyms. This is because the Electric gym is currently working hard to get people in to Pokémon batteling. We are giving players assignments to improve their knowledge about Pokémon, for example in abillities and items. How to use them. 

This is helping them in a better way, in my opinion then making it easier for them to get badges. NPC's are just braindead and to easy. You can set their difficulty, but it's not so hard to defeat them. And they will not represent how the "real" gymleaders battle and how they give advice.

Okaay now you lost against the NPC leader, what did you do wrong? Well I don't know and I can't give advice towards that. The fun part of the Gym's in Stratos is that people do have to think of their teams and how to use them. And when they finally defeat me as electric leader.... I'm so happy for them! Because they understand battleing now! Not for a full 100% but approx 60%. That's where the poison gym comes in, they can build on how we in the electric gym set the base.

TL:DR, I don't like the idea of easy mode gyms.

**Comment by Archenemis** [comment]
Thank you for the response and feedback. The intention of the Easy Mode Gym isn't to give them a free pass though.

In fact they won't be earning the badge at all.. If anything.. think of the Easy Mode Gym Leader as more of a "Gym Assistant" They will give a slight reward for beating them.. But they won't give you a badge. And you still have to beat the Electric Gym Leader before you can go to the poison gym leader. (To Clarify. You must defeat one of the Human Gym Leaders before you can progress further) If anything the Easy Mode is intended to be more of a Training Dummy.

There's also only so many Gym Leaders and 6 hours in between the times that they can usually challenge the actual Gym Leaders. With the Easy Mode being an option for those who aren't as competitive or are a bit more casual. (Or just wanting to work out a few ideas or test a team against someone that they won't feel is judging them or their team.)

This isn't a replacement for the Gym Challenge. It's just more of a stepping stone and to help give players a little more to do and kind of gives the players some extra tools they can use.. Plus the Gyms are quite well designed.. It's nice to get to show them off.

My only real concern is that players who don't read or listen.. Might assume that the NPC's for the Gyms are the actual leaders and get confused as to why they don't get a badge. Or get confused as to why they can't challenge the next Leader.. Things like that I could see being a problem.

To Shorten it down: They won't get badges from this. This is more for practice. It helps to encourage those who might be shy to step into the Gym Scene. And it gives people something that is in the Gym at all times that they can go to at least test the team they've got prepared for the electric gym out on. If they can't beat that NPC. They definitely can't beat the player. To me, this is a thousand time more favorable than continuously nerfing the Electric gym to make it easier. 5 Pokes is an extremely harsh nerf. Lowering their IV's and EVs would just be even worse. I'd rather try this out and see how it works out than resort to anymore nerfs on Gyms.

**Comment by ThaGreenDuck** [comment]
> In fact they won't be earning the badge at all.. If anything.. think of the Easy Mode Gym Leader as more of a "Gym Assistant" They will give a slight reward for beating them.. But they won't give you a badge. And you still have to beat the Electric Gym Leader before you can go to the poison gym leader. (To Clarify. You must defeat one of the Human Gym Leaders before you can progress further) If anything the Easy Mode is intended to be more of a Training Dummy.

This mighy be true... but you point out the concern in your statement:

> My only real concern is that players who don't read or listen.. Might assume that the NPC's for the Gyms are the actual leaders and get confused as to why they don't get a badge. Or get confused as to why they can't challenge the next Leader.. Things like that I could see being a problem

Training dummy's can be implemented without being called "gymleaders". You can constanly rechallenge them but they do not give any rewards. Then you can make the difference between a hard or easy training dummy. That's a way people can start to improve their battle skills. But you make an other interesting point in your answer as well:

> To me, this is a thousand time more favorable than continuously nerfing the Electric gym to make it easier. 5 Pokes is an extremely harsh nerf. Lowering their IV's and EVs would just be even worse. I'd rather try this out and see how it works out than resort to anymore nerfs on Gyms. 

I do not agree to this point. From my standing we have to look at it like this, the Electric gym is the first gym people do have to encounter. With this in mind it should be the easiest gym to beat. Currently it's on of the hardest gym to beat, due to the fast mons and low level cap. That people have to deal with. I know as matter of fact that Sahv is working her ass of to get a good balanced way to let people deal with the electric gym. 

5 mons isn't a extremly harsh nerf in my opinion, this is because the Electric gym still has tons of options to go with. IV/EV lowering, that's also a good option to go with, why? You would ask... like I said a little before. The Electric should be the easiest gym to be beaten, and people who just started Pixelmon.... will not have quick access to high IV's mons or the correct nature of mons. I have completed the gyms without the high IV mons or correct nature. For example I used a Mega Gyarados with only 55% iv's, no 31 iv stats and the nature was Bashful. So IV's and natures aren't as important to the gyms in my opinion. So nerving the starting gyms is a good option.

TL:DR, training dummy's yes, but not as gym leaders. No rewards for the dummy's as well. Maybe for the hardest dummy? And for gyms, the level has to rise little by little.

**Comment by Archenemis** [comment]
Yeah. I think it’s only fair to present the possible negative outcomes to an idea. It helps to bring out possibilities to fixing it and helps to fairly assess if it’s good for everyone.

 Stratos also has a reputation for being player ran only. So it would be changing things a tiny bit. I do get what you’re saying in difficulty needing to rise slowly, but that would likely mean needing to nerf all the gyms and taper them a little. And electric gym seems to be the most draining. If people don’t like the idea, it doesn’t need to be done, I just felt it could be a useful tool and help to give those who are waiting for a gym leader something to practice and sharpen their team on.

But.. are you saying it would be more acceptable to you if we called them “Gym assistants” ? Also.. I do know the gyms are being worked on, my suggestion is more that perhaps the challengers should try and meet the leaders halfway. A lot of the challengers come to the gyms unprepared and having high expectations to just easily win. This creates frustration and gets people flustered when they lose, even more so if they were having to sit idle until their match. If the dummies give them trouble.. or they have to resort to cheap tactics to beat it.. they don’t really stand a chance against the leader.

I also was under the impression that the gyms were an introduction to competitive play and are supposed to be difficult.

My main goal in my ideas is to try and help the server. So I do hope that this idea isn’t taken to say that things are bad or that what all is being done so far isn’t appreciated. It is. And I hope no one feels insulted with this suggestion.

Just stating that from my analysis the biggest problem isn’t the gyms.

**Comment by Archenemis** [comment]
Something else I kind of have been concerned about. Is the level of team sharing, level of strat sharing and Pokémon being mentioned about in the gyms.

I think players in general are approaching the gyms perhaps in the wrong mindset. To go along with the easy mode thing.. or perhaps even to just be added on its own. I feel it may be important to mention somewhere.. whether it’s in the rules, or the /gyms rules or somewhere very visible in spawn. But I think it may help cut down on toxicity and help people understand what the gyms are about if a small statement was made about them. Gym leaders are human and are capable of mistakes, getting upset, frustration and all the like. They do not gain pleasure from tormenting you. They do not get paid for tormenting you. Heir goal is also not to torment you. But think it should be said:

“The gym leaders want to see you succeed. If you are having trouble,feel free to talk to the gym leaders and get advice from them. They will coach you on how to build a better team and how to improve as a trainer. They are your rivals through the gyms, but that doesn’t mean they can’t be friendly too”

I feel like having this be said, may help players to feel they can approach the gym leaders a bit more and get actual, helpful, non-bannable advice from them, since most gym leaders know what is saying too much and what is helpful simply helpful advice. It may also help players to realize the things I mentioned above the quoted statement, that the gyms are fun and they are meant to be enjoyed. It may even help players to realize that the rewards and items are nice from the gyms, but that the journey and experience they get from building their own team and learning to play with the team they’ve built is the real reward. Perhaps some things could be added or removed from what I said, but overall would love to hear what people think of having this sort of “note” tacked in somewhere

I just want to help players and staff alike to make stratos even more friendly than it already is.

Oh.. one more thing. I personally don’t think it would be a huge issue if the gym leaders received some form of compensation in-game. Many of them deal with getting trash talked, treated toxic, made fun of, picked on, yelled at, called cheats, deal with shared teams, people yelling out their teams, or using other people’s pokemon against them. I feel a little in-game alleviation would help to make swallowing some of the insults and accusations that come their way a bit easier. Whether it’s monetary or perhaps through another means. Just once again, desiring to make the server a bit more enjoyable for everyone
